cinema/views.py

```diff
@@ -126,6 +123,29 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    # only for documentation purposes
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                type=str,
+                description="Filter movie by title"
+            ),
+            OpenApiParameter(
+                name="genres",
+                type={"type": "list", "items": {"type": "number"}},
```

Comment:
You are not using "list" here. Here you need to specify the type that is used.

Reply:
i think i specify the type in items, according to the video tutorial and everything works fine, i've tested, maybe there is the other way of specifying the type